commands/geoadd.md

```diff
@@ -1,47 +1,87 @@
 Adds the specified geospatial items (latitude, longitude, name) to the specified
-key. Data is stored into the key as a sorted set, in a way that makes it possible to later retrieve items using a query by radius with the `GEORADIUS` or `GEORADIUSBYMEMBER` commands.
+key.
+Data is stored into the key as a sorted set, in a way that makes it possible
+to later retrieve items using a query by radius with the `GEORADIUS` or
+`GEORADIUSBYMEMBER` commands, or use `GEOSEARCH` searching within rectangular
+areas.
 
-The command takes arguments in the standard format x,y so the longitude must
-be specified before the latitude. There are limits to the coordinates that
-can be indexed: areas very near to the poles are not indexable. The exact
-limits, as specified by EPSG:900913 / EPSG:3785 / OSGEO:41001 are the following:
+The command takes arguments in the standard format x,y so the longitude must be
+specified before the latitude.
+There are limits to the coordinates that can be indexed: areas very near to the
+poles are not indexable.
+The exact limits, as specified by EPSG:900913 / EPSG:3785 / OSGEO:41001 are the
+following:
 
 * Valid longitudes are from -180 to 180 degrees.
 * Valid latitudes are from -85.05112878 to 85.05112878 degrees.
 
-The command will report an error when the user attempts to index coordinates outside the specified ranges.
+The command will report an error when the user attempts to index coordinates
+outside the specified ranges.
 
-**Note:** there is no **GEODEL** command because you can use `ZREM` in order to remove elements. The Geo index structure is just a sorted set.
+**Note:** there is no **GEODEL** command because you can use `ZREM` in order to
+remove elements.
+The Geo index structure is just a sorted set.
 
-How does it work?
----
+## GEOADD options
+
+GEOADD supports a list of options, specified after the name of the key and
+before the first longitude argument.
+Options are:
+
+* **XX**: Only update elements that already exist.
+  Never add elements.
+* **NX**: Don't update already existing elements.
+  Always add new elements.
+* **CH**: Modify the return value from the number of new elements added, to the
+  total number of elements changed (CH is an abbreviation of _changed_).
+  Changed elements are **new elements added** and elements already existing for
+  which **the score was updated**.
+  So elements specified in the command line having the same score as they had in
```

Comment:
the reference to "score" here is invalid. should be "coordinates"?

Reply:
yeah, thanks